x-pack/plugins/observability_solution/dataset_quality/public/hooks/use_degraded_fields.ts

```diff
@@ -62,17 +62,48 @@ export function useDegradedFields() {
     return sortedItems.slice(page * rowsPerPage, (page + 1) * rowsPerPage);
   }, [data, sort.field, sort.direction, page, rowsPerPage]);
 
-  const isLoading = useSelector(service, (state) =>
+  const isDegradedFieldsLoading = useSelector(service, (state) =>
     state.matches('initializing.dataStreamDegradedFields.fetching')
   );
 
+  const closeDegradedFieldFlyout = useCallback(
+    () => service.send({ type: 'CLOSE_DEGRADED_FIELD_FLYOUT' }),
+    [service]
+  );
+
+  const openDegradedFieldFlyout = useCallback(
+    (fieldName: string) => {
+      if (expandedDegradedField === fieldName) {
+        service.send({ type: 'CLOSE_DEGRADED_FIELD_FLYOUT' });
+      } else {
+        service.send({ type: 'OPEN_DEGRADED_FIELD_FLYOUT', fieldName });
+      }
+    },
+    [expandedDegradedField, service]
+  );
+
+  const degradedFieldValues = useSelector(service, (state) =>
+    state.matches('initializing.initializeFixItFlow.ignoredValues.done')
```

Comment:
shouldn't be always `state.context.degradedFieldValues`?
if it's not done I'd think the value is `undefined`

Reply:
Now that you mentioned it, it makes sense.
I guess i was getting too cautious 😆 

Will change it